restapi/stories.go

```diff
@@ -190,6 +189,10 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+
+	// We set the story to be private when it is initially uploaded
+	story.IsPublic = false
```

Comment:
What is the point of having a "is_public" json field if you're going to set it to false anyways when you first upload it?

Reply:
It'll be private on upload, and only set to public when we publish the map.